### ISSUE TITLE
pillow 7.0.0 pkgs for python 3.7 and 3.8 not available for ppc64le

### ISSUE THREAD

**Issue by hartb** [ContinuumIO/anaconda-issues#11566]
### Actual Behavior

`pillow` `7.0.0` is available in the Anaconda repos now.

For x86, builds are available for all of python 3.6, 3.7, and 3.8:

```
$ conda search pillow
Loading channels: done
...
pillow                         7.0.0  py36hb39fc2d_0  pkgs/main
pillow                         7.0.0  py37hb39fc2d_0  pkgs/main
pillow                         7.0.0  py38hb39fc2d_0  pkgs/main
```

But for `ppc64le`, only the python 3.6 package is available:

```
$ conda search pillow
Loading channels: done
...
pillow                         7.0.0  py36haac5956_0  pkgs/main
```

Confimed in the actual repo as well: https://repo.continuum.io/pkgs/main/linux-ppc64le/

### Expected Behavior

Please build the python 3.7 and 3.8 packages of pillow 7.0.0 for ppc64le as well.

### Steps to Reproduce

See above.

##### Anaconda or Miniconda version:

4.7.12

##### Operating System:

Linux on x86 and ppc64le

##### `conda info`

<details>

```
$ conda info  

     active environment : wmlce
    active env location : /opt/anaconda/envs/wmlce
            shell level : 2
       user config file : /home/pwrai/.condarc
 populated config files : /home/pwrai/.condarc
          conda version : 4.7.12
    conda-build version : not installed
         python version : 3.7.4.final.0
       virtual packages : 
       base environment : /opt/anaconda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-ppc64le
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-ppc64le
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/anaconda/pkgs
                          /home/pwrai/.conda/pkgs
       envs directories : /opt/anaconda/envs
                          /home/pwrai/.conda/envs
               platform : linux-ppc64le
             user-agent : conda/4.7.12 requests/2.22.0 CPython/3.7.4 Linux/4.14.0-115.14.1.el7a.ppc64le ubuntu/18.04.3 glibc/2.27
                UID:GID : 2051:2051
             netrc file : None
           offline mode : False
```

</details>

**Comment by jjhelmus** [comment]
pillow 7.0.0 packages are now available for Python 3.6, 3.7 and 3.8 for the linux-ppc64le platform.  I'm not certain why the 3.7 and 3.8 packages were missing.

**Comment by hartb** [comment]
Confirmed; thank you very much!
